Makefile

```diff
@@ -315,13 +315,74 @@ src/newrelic/infinite_tracing/com_newrelic_trace_v1/v1.pb.go: protocol/infinite_
 #
 
 .PHONY: integration
-integration: Makefile daemon lasp-test-all
+integration: Makefile daemon lasp-test-all integration-events-limits
 	for PHP in $${PHPS:-8.1 8.0 7.4 7.3 7.2 7.1 7.0 5.6 5.5}; do \
           echo; echo "# PHP=$${PHP}"; \
 	  env NRLAMP_PHP=$${PHP} bin/integration_runner $(INTEGRATION_ARGS) || exit 1; \
 	  echo "# PHP=$${PHP}"; \
 	done
 
+#
+# Agent event limits integration testing
+#
+# Because of how the integration_runner connects to the collector once before
+# running any tests we have to tell the runner the value the agent would
+# have passed to it for each test.  This means these tests are not testing
```

Comment:
```suggestion
# Because of how the integration_runner connects to the collector (once before
# running any tests) we have to tell the runner the value the agent would
# have passed to it for each test.  This means these tests are not testing
```